Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js

```diff
@@ -0,0 +1,31 @@
+const game = {
+  lives: 3,
+  coins: 0,
+  get points() {
+    let point = this;
+    point.coins = 2;
+
+    console.log(`points= ${this.coins * 10}`);
+  },
+  playerDies: function () {
+    let chance = this;
+    if (this.lives !== 0) {
+      chance.lives = this.lives - 1;
```

Comment:
this.lives>0 , this should be the condition

Reply:
Fixed the changes. please review.